app-frontend/packs/components/homePage/main/MainApp.vue

```diff
@@ -0,0 +1,132 @@
+                <feOffset dy="4"/>
+                <feGaussianBlur stdDeviation="2"/>
+                <feComposite in2="hardAlpha" operator="arithmetic" k2="-1" k3="1"/>
+                <feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/>
+                <feBlend mode="normal" in2="shape" result="effect1_innerShadow_72_72"/>
+                </filter>
+                <filter id="filter1_i_72_72" x="376.971" y="298.92" width="49.0398" height="57.7955" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+                <feFlood flood-opacity="0" result="BackgroundImageFix"/>
+                <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape"/>
+                <feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+                <feOffset dy="4"/>
+                <feGaussianBlur stdDeviation="2"/>
+                <feComposite in2="hardAlpha" operator="arithmetic" k2="-1" k3="1"/>
+                <feColorMatrix type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/>
+                <feBlend mode="normal" in2="shape" result="effect1_innerShadow_72_72"/>
+                </filter>
+                <filter id="filter2_d_72_72" x="46.5994" y="392.227" width="655.003" height="81.7273" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+                <feFlood flood-opacity="0" result="BackgroundImageFix"/>
+                <feColorMatrix in="SourceAlpha" type="matrix" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/>
+                <feOffset dy="6"/>
+                <feGaussianBlur stdDeviation="2"/>
+                <feComposite in2="hardAlpha" operator="out"/>
+                <feColorMatrix type="matrix" values="0 0 0 0 0.540688 0 0 0 0 0.495521 0 0 0 0 0.8375 0 0 0 1 0"/>
+                <feBlend mode="normal" in2="BackgroundImageFix" result="effect1_dropShadow_72_72"/>
+                <feBlend mode="normal" in="SourceGraphic" in2="effect1_dropShadow_72_72" result="shape"/>
+                </filter>
+            </defs>
+        </svg>
+        <div class="bl-main__subtitle">
+            Ready to get started create banners with no code content? Welcome to our app!
+        </div>
+    </div>
+    <div v-if="this.$store.state.banners.isLoader">
+        <LoaderVue/>
+    </div>
+</template>
+
+<script>
+    import Card from './card/Card.vue'
+    import LoaderVue from '../../sharedComponents/LoaderVue.vue'
+    
+    export default {
+        name: 'MainApp',
+        data() {
+            return {
+                responseData: Boolean,
+            }
+        },
+        components: {
+            Card,
+            LoaderVue,
+        },
+        methods: {
+            isTrue() {
+                if(this.$store.state.banners.bannersData) {
```

Comment:
`this.responseData = !!this.$store.state.banners.bannersData`
should do the same thing as if else statement